src/backend/api/Fusion.Resources.Api/Controllers/Utilities/UtilitiesController.cs

```diff
@@ -0,0 +1,140 @@
+﻿using Fusion.Integration.Configuration;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace Fusion.Resources.Api.Controllers.Utilities
+{
+    [Authorize]
+    [ApiController]
+    public class UtilitiesController : ResourceControllerBase
+    {
+        private readonly IHttpClientFactory httpClientFactory;
+        private readonly IFusionTokenProvider tokenProvider;
+        private readonly IConfiguration configuration;
+        private readonly IOptions<FusionIntegrationOptions> fusionOptions;
+
+        public UtilitiesController(IHttpClientFactory httpClientFactory, IFusionTokenProvider tokenProvider, IConfiguration configuration, IOptions<FusionIntegrationOptions> fusionOptions)
+        {
+            this.httpClientFactory = httpClientFactory;
+            this.tokenProvider = tokenProvider;
+            this.configuration = configuration;
+            this.fusionOptions = fusionOptions;
+        }
+
+        [HttpPost("/utilities/parse-spreadsheet")]
+        public async Task<ActionResult<ExcelConversion>> ValidateContractorImportSpreadsheet([FromForm]ConvertSpreadsheetRequest request)
+        {
+            if (request == null)
+                return FusionApiError.InvalidOperation("MissingBody", "Could not locate any body payload");
+
+            var url = $"https://pro-f-utility-{fusionOptions.Value.Environment}.azurewebsites.net";
+            var token = await tokenProvider.GetApplicationTokenAsync();
+            var client = httpClientFactory.CreateClient();
+            client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("bearer", token);
+            client.BaseAddress = new Uri(url);
+
+
+            if (string.IsNullOrEmpty(url))
+                throw new InvalidOperationException("Missing configuration for fusion utility function");
+
+
+            using (var streamContent = new StreamContent(request.File!.OpenReadStream()))
+            {
+                streamContent.Headers.ContentType = new MediaTypeHeaderValue("application/octet-stream");
+                streamContent.Headers.ContentLength = request.File.Length;
+
+                var response = await client.PostAsync("/api/excel-json-converter", streamContent);
+
+                var content = await response.Content.ReadAsStringAsync();
+
+                if (response.IsSuccessStatusCode)
+                    return JsonConvert.DeserializeObject<ExcelConversion>(content);
+
+                throw new InvalidOperationException($"Parser function returned non-successfull response ({response.StatusCode}).");
```

Comment:
Same goes here, I guess.

Reply:
Same response i guess :)
I guess this could be a bad request, as the spread sheet could be invalid etc, but the effort to analyse and respond differently does not really outweigh the benefit.

Reply:
Agreed. See discussion in previous comment.